eta/modules/sample_videos.py

```diff
@@ -56,8 +56,8 @@ class DataConfig(Config):
         input_path (eta.core.types.Video): The input video
 
     Outputs:
-        output_path (eta.core.types.ImageSequence): The output sampled video
-            frames
+        output_path (eta.core.types.VideoFile):
+            the path to the sampled video
```

Comment:
Changing the type of an established module's output is tricky, because now you've broken any existing pipelines or configs that counted on this generating an image sequence.

We were able to get away with this with the tracking modules because they were brand new and you knew all the configs to change, but here we need to make sure we retain backwards compatibility.

My recommendation was to add *another* output that optionally allows the video file to be written, not image frames. This is the easiest solution because its backwards compatible. But `output_path` is a bad name in this case; `output_impath` would be better...

Reply:
Right, I didn't think about that. However, changing `output_path` to `output_impath` would require changing the config files for other projects that use `sample_videos.py`. I'm fine with doing it, but I just want to check with you first. Do you think we should do this?

Reply:
Sorry I missed this comment earlier- yeah let's go ahead and use the best names we can. *I'll take care of it though*